### ISSUE TITLE
Commenting Blank Lines in Multi-Line Block

### ISSUE THREAD

**Issue by aoozdemir** [microsoft/vscode#65109]
I don't know if this is a bug or a feature. The same question asked on SO, [here](https://stackoverflow.com/questions/52919515/commenting-blank-lines), but there was no answer and I couldn't find an issue that referred to this after searching with obvious keywords. Apologize in advance if this has been dealt with already.

Problem is basically this.
If I have a method as such and highlight the whole block and comment out it with `CTRL + /`, I'll get this;
```ruby
# def some_ruby_method
#   puts "line"
#   puts "line"

#   puts "line"
# end
```
It will skip the blank line, whereas it should behave as follows;
```ruby
# def some_ruby_method
#   puts "line"
#   puts "line"
# 
#   puts "line"
# end
```

I also do remember this behaviour was exactly like the latter before but don't know when and on which version it changed.

This really bugs me to my bones and I wonder what is the reason behind this design or if there is a settings for that. And really wonder what the community thinks about it.

**Comment by peterflynn** [comment]
Looks like this was fixed via a new preference in v1.48 (July 2020): see PR #93160 and duplicate issue #88480.

**Comment by aoozdemir** [comment]
I guess I made my peace with it at some point :) But yeah, the option is there and now we have a choice to select which way we want. Thanks for the comment, closing this issue now.
